config/govuk_web_banners/global_banners.yml

```diff
@@ -0,0 +1,24 @@
+# Example usage of adding a banner to the global_banners list
```

Comment:
I think this was already quite well documented in a previous commit, would it save keeping two things up to date if this linked to that file instead?

Reply:
I've updated the config files to just point to the README.md